store/store.go

```diff
@@ -139,6 +145,23 @@ func NewBlockStore(db dbm.DB, options ...BlockStoreOption) *BlockStore {
 	return bStore
 }
 
+func (bs *BlockStore) addCaches() {
+	var err error
+	// err can only occur if the argument is non-positive, so is impossible in context.
+	bs.blockCommitCache, err = lru.New[int64, *types.Commit](100)
```

Comment:
I know that it is somehow arbitrary, but why 100? :D

Reply:
It was very much arbitrary, it has to be bigger than like 3 for live consensus + peer catchup. Was hoping that if its a bigger value, we wouldn't get too much thrashing while also serving blocksync